bento-platform/bento_wes
# Features/cbioportal

## New Features:
- cbioportal related:
  - vep cache initialization. This can be triggered from BentoV2 make file (`make init-wes-cache`)
  - handle analysis workflows

## Code Changes:
- Workflow parameters handling has been augmented with the possibility to interpolate some parameters from config.
- Auth tokens used within workflows can be declared in the workflow metadata and injected in the workflow parameters.
- DRYed some code in relation with authentication tokens.

## How to test:
- BentoV2 must be updated to run the cache init script (branch `feature/cbioportal`)
- Katsu must be updated to run the analysis workflow (branch `feature/cbio_export_workflow`)
- Bento-web must be updated to test analysis and export workflows (branch `cbio_export`)

- Make sure that the previous workflows (e.g. phenopackets, vcf files ingestion) are still running correctly
- The procedure for the vcf2maf + export workflows is described here: https://github.com/bento-platform/bentoV2/pull/47
